test/e2e_node/conformance_test.go

```diff
@@ -0,0 +1,116 @@
+/*
+Copyright 2015 The Kubernetes Authors All rights reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package e2e_node
+
+import (
+	"fmt"
+	"time"
+
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+	"k8s.io/kubernetes/pkg/api"
+	"k8s.io/kubernetes/pkg/api/errors"
+	client "k8s.io/kubernetes/pkg/client/unversioned"
+)
+
+var _ = Describe("Kubelet", func() {
+	var cl *client.Client
+	var terminateCase ConformanceContainer
+	var invalidImageCase ConformanceContainer
+
+	BeforeEach(func() {
```

Comment:
Instead of creating both pods for each Context, why not create only the client init here, and then move terminatedCase to the terminated Context and invalidImageCase to the invalidImage Context